### DISCUSSION TITLE
Choice of the platform for the future Krita Ai diffusion user group

### DISCUSSION THREAD

**Discussion by minsky91** [Acly/krita-ai-diffusion#877] [Polls]
Okay, it seems the idea of starting a dedicated Krita Ai diffusion user group has been well received here, as indicated by the 5 likes so far - for this github group this should suffice, I think. Let’s now decide which social media platform might be suited the best for hosting such a group.

**Comment by dkqjhoifh** [comment]
The krita ai plugin is great, but krita itself has a very small user base, photoshop has been popularized in the field of art, which makes it very difficult for other painting software to popularize, if you want to build a community, you want the community to be lively, then, only wait for krita to replace ps or krita ai to support Photoshop, as far as I know, there seem to be people in China where I am trying to integrate the krita ai plugin into ps, But I think it's best if Acly does the work!

**Reply by dkqjhoifh** [reply]
For professional artists who are accustomed to PS, integrating Krita AI into PS is much easier to accept than taking a long time to change their habits and learn a new drawing software.

**Reply by parella007** [reply]
Hello dkqjhoifh, I understand what you mean with photoshop being popular and all that but I think what valery is trying to do is to gather (even if it is small) a community of people who are using this ai plugin to share experiences and in some way promote it to become more popular. Which I think is a wonderful idea.

I am an architect, and i use professional 3d and 2d software everyday (including photoshop and its generative fill which i dont like at all), but this Acly-krita-ai-diffusion has rocked my world, I am still a complete newby and I have already made some progress with architectural renders with this plugin, and recommended it to some friends who are all as amazed as myself, creating such a user group could help us all, a lot.

**Reply by minsky91** [reply]
Hi parella007, glad to see your support for the idea. Is there any chance you could also consider to be a co-moderator of the group?

**Reply by parella007** [reply]
Hi, I would be happy to help, although I don't know if this would mean anything since I have never used Discord and I am still very new to this.
Anyway i will surely join and participate.
